### PR TITLE
Setting up react-native-navigation

### PR DESCRIPTION
**Pull request by eduardomoroni**
Setting up react-native-navigation

https://wix.github.io/react-native-navigation/#/

### CODE REVIEW

File: src/Themes/Navigation.ios.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    label: 'SignIn',
+    screen: 'example.Types',
+    icon: require('../Assets/Images/swap.png'),
+    title: 'Sign In Screen'
+  }, {
+    label: 'SignIn',
+    screen: 'example.Types',
+    icon: require('../Assets/Images/swap.png'),
+    title: 'Sign In Screen'
+  }
```
Comment (taniadgv) [inline src/Themes/Navigation.ios.js]: I'm not sure why we register the SignIn here twice 😕

**Comment by eduardomoroni** [comment]
It was just lazyness. I should've used RideList screen and SignIn Screen.
At this point though, I didn't know how to use RideList as Screen with
properties.

Em dom, 2 de jul de 2017 11:36 AM, Tania Gonzales <notifications@github.com>
escreveu:

File: src/Themes/Navigation.ios.js (hunk 1 above)
Comment (eduardomoroni) [inline src/Themes/Navigation.ios.js]: Next PR will solve this,
image
image
image

Comment (eduardomoroni) [inline src/Themes/Navigation.ios.js]: Sorry folks, I was just coding fast we'll need to refactor together.

**Pull request merged**